README.md

```diff
@@ -0,0 +1,60 @@
+## Quantitative cardiac image analysis data module: `qcardia-models`
+
+A PyTorch based library to build and handle medical imaging model pipelines. Can be used to quickly get highly customizable U-Net or EncoderMLP models for deep learning purposes. Currently supported models:
+| # | Name | Dimensionality | Use case |
+|-|-|-|-|
+| 1 | U-Net   | 2D | Dense predictions (e.g. segmentation)
+| 2 | EncoderMLP | 2D | Encoder pretraining tasks
+
+
+### Installation
+#### Environment and PyTorch
+It is recommended to make a new environment (tested for Python 3.11.9) and first installing PyTorch and checking GPU availability. It is recommended to install the PyTorch version the package was tested for ([PyTorch 2.3.1](https://pytorch.org/get-started/previous-versions/#v231)), which should limit warnings or unexpected behaviours. Alternatively, installation instructions for the latest stable PyTorch version can also be found in [their "get started" guide](https://pytorch.org/get-started/locally/).
+
+#### Stable version of `qcardia-models`
+Install from GitHub using pip to get the latest version:
+```
+pip install git+https://github.com/q-cardIA/qcardia-models
+```
+
+Or if you want a specific version, include the release version in the link, e.g.:
+```
+pip install git+https://github.com/q-cardIA/qcardia-models@v1.0.0
+```
+
+Available versions can be found in the [releases tab](https://github.com/q-cardIA/qcardia-models/releases), where the release [tags](https://github.com/q-cardIA/qcardia-models/tags) are used in the install command.
+
+#### Editable version of `qcardia-models`
+You can install a local copy in `editable` mode to make changes to the package that instantly get reflected in your environment. After getting a local copy (download/clone/fork), install using:
+```
+pip install -e "path/to/qcardia-models"
+```
```

Comment:
You need to have a recent version of pip for this to work. Old versions required a `setup.py`.
So I like to add also the disclaimer:

(It may be neccessary to first upgrade pip using `python3 -m pip install --upgrade pip`).